docs/application/dotnet/guides/nui/xaml/introduction-with-example.md

```diff
@@ -0,0 +1,147 @@
+# Example
+You can use XAML to define the layout of your user interface. To define the layout, follow these steps:
+1. Create a XAML file
+2. Load the XAML file
+3. Put the XAML file in the appropriate file directory
+
+> **Note**
```

Comment:
Suggest rephrasing to:

Reply:
@dongsug-song,
Sorry for the miss. Suggest rephrasing to:

WPF XAML provides a template to create an application easily, but currently NUI XAML does not have a XAML application template. However, NUI XAML will be opened and released at **Tizen.NET API7** version.